pkg/plugins/golang/v2/scaffolds/internal/templates/main.go

```diff
@@ -114,9 +114,11 @@ const (
 		os.Exit(1)
 	}
 `
-	webhookSetupCodeFragment = `if err = (&%s.%s{}).SetupWebhookWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create webhook", "webhook", "%s")
-		os.Exit(1)
+	webhookSetupCodeFragment = `if os.Getenv("ENABLE_WEBHOOKS") != "false" {
+		if err = (&%s.%s{}).SetupWebhookWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create webhook", "webhook", "%s")
+			os.Exit(1)
+		}
```

Comment:
This plugin is deprecated for. along period so we no longer add changes on it. 
Could you please remove the change here?

Reply:
removed